invenio_app_rdm/cli.py

```diff
@@ -40,3 +42,20 @@ def create_static_pages(force):
     Pages().run(force=force)
 
     click.secho("Created static pages!", fg="green")
+
+
+@rdm.command("rebuild-indices")
```

Comment:
```suggestion
@rdm.command("rebuild-all-indices")
```
to avoid name confusion with existing rebuild-index command